### PR TITLE
Fixed check on registered tokens before removing

### PR DESCRIPTION
**Pull request by tomiir**

### CODE REVIEW

File: src/PlugWallet/index.ts [hunk 1]
```diff
@@ -237,12 +237,12 @@ class PlugWallet {
   };
 
   public removeToken = (tokenId: string) => {
-    if(!(tokenId in this.registeredTokens))
+    if(!(Object.keys(this.registeredTokens).includes(tokenId))) {
```
Comment (aleacevedo) [inline src/PlugWallet/index.ts:240]: This change should be the same, the operator in check into the keys

File: src/PlugWallet/index.ts [hunk 2]
```diff
@@ -237,12 +237,12 @@ class PlugWallet {
...
       return Object.values(this.registeredTokens);
-
-    const {[tokenId]: removedToken, ...newTokens} = this.registeredTokens;
+    }
+    const { [tokenId]: removedToken, ...newTokens } = this.registeredTokens;
     this.registeredTokens = newTokens;
-    this.assets = this.assets.filter(asset => asset.canisterId !== tokenId);
+    this.assets = [...this.assets.filter(asset => asset.canisterId !== tokenId)];
```
Comment (aleacevedo) [inline src/PlugWallet/index.ts:245]: This change also should be the same

File: src/PlugWallet/index.ts (hunk 1 above)
Comment (rocky-fleek) [reply src/PlugWallet/index.ts:240]: it was failing that way

File: src/PlugWallet/index.ts (hunk 2 above)
Comment (rocky-fleek) [reply src/PlugWallet/index.ts:245]: this is a Just in case lol

**Pull request merged**